scopes/scope/scope/scope.main.runtime.ts

```diff
@@ -351,8 +353,39 @@ export class ScopeMain implements ComponentFactory {
   private localAspects: string[] = [];
 
   async loadAspects(ids: string[], throwOnError = false): Promise<void> {
+    const scopeManifests = await this.getManifestsGraphRecursively(ids, throwOnError);
+    await this.aspectLoader.loadExtensionsByManifests(scopeManifests);
+  }
+
+  async getManifestsGraphRecursively(ids: string[], throwOnError = false): Promise<ManifestOrAspect[]> {
```

Comment:
@davidfirst you might have the same ids multiple time during the graph
a -> b -> c -> d
f -> b -> c -> d
you will calculate the b->c->d chain twice. 
Need to have some kind of visited pattern.
(we should probably apply the same in the `resolveRequireableExtensionManifestDepsVersionsRecursively` )

Reply:
Done. I added `visited()` param.